compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/QueryDesugar.java

```diff
@@ -337,6 +326,27 @@ BLangStatementExpression desugar(BLangQueryExpr queryExpr, SymbolEnv env,
         return streamStmtExpr;
     }
 
+    private boolean isXml(BType type) {
```

Comment:
Wouldn't this be more useful if we have it in `Types`?

Reply:
+1. You mean into `types` right?